custom/ewsghana/reports/__init__.py

```diff
@@ -168,3 +171,29 @@ def is_reporting_type(self):
             if not location_type.administrative
         ]
         return sql_location.location_type in reporting_types
+
+    @property
+    def export_table(self):
+        r = self.report_context['reports'][0]['report_table']
+        return [self._export_table(r['title'], r['headers'], r['rows'])]
+
+    def _export_table(self, export_sheet_name, headers, formatted_rows, total_row=None):
+        def _unformat_row(row):
+            return [col.get("sort_key", col) if isinstance(col, dict) else col for col in row]
+
+        table = headers.as_export_table
+        rows = [_unformat_row(row) for row in formatted_rows]
+        for row in rows:
+            row[1] = row[1][:row[1].index('<')]
```

Comment:
this was mainly what i was referring to above - and i was suggesting putting a comment in the code so future people understand, not just in the PR